src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -90,6 +101,14 @@ public void BuildReleaseArm64 ([Values (false, true)] bool forms)
 					? GetLinkedPath (b, true, depsFilename)
 					: Path.Combine (proj.Root, b.ProjectDirectory, depsFilename);
 				FileAssert.Exists (depsFile);
+
+				const int ApkSizeThreshold = 50 * 1024;
+				const int AssemblySizeThreshold = 50 * 1024;
+				var apkFile = Path.Combine (Root, b.ProjectDirectory, proj.OutputPath, proj.PackageName + "-Signed.apk");
+				var apkDescPath = Path.Combine (Root, b.ProjectDirectory, apkDescFilename);
+				var apkDescReferencePath = Path.Combine (Root, b.ProjectDirectory, apkDescReference);
+				var (code, stdOut, stdErr) = RunApkDiffCommand ($"-s --save-description-2={apkDescPath} --test-apk-size-regression={ApkSizeThreshold} --test-assembly-size-regression={AssemblySizeThreshold} {apkDescReferencePath} {apkFile}");
```

Comment:
This appears to be failing on Windows: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4413244&view=ms.vss-test-web.build-test-results-tab&runId=18498752&resultId=100107&paneView=debug

```
System.ComponentModel.Win32Exception : The system cannot find the file specified.

   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at Xamarin.Android.Build.Tests.BaseTest.RunProcessWithExitCode(String exe, String args) in /Users/builder/azdo/_work/1/s/xamarin-android/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs:line 277
   at Xamarin.Android.Build.Tests.BaseTest.RunApkDiffCommand(String args) in /Users/builder/azdo/_work/1/s/xamarin-android/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs:line 244
   at Xamarin.Android.Build.Tests.BuildTest.BuildReleaseArm64(Boolean forms) in /Users/builder/azdo/_work/1/s/xamarin-android/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs:line 110

```

Reply:
I have added a fallback to run apkdiff from known location - `~/dotnet/tools/`, in case the PATH wasn't updated properly.